error-codes.md

```diff
@@ -532,7 +532,7 @@ TiDB is compatible with the error codes in MySQL, and in most cases returns the
 
 * Error Number: 9001
 
-    The complete error message: `ERROR 9001 (HY000): PD Server Timeout`
+    The complete error message: `ERROR 9001 (HY000): PD server timeout`
```

Comment:
@yahonda Could you please update "TiKV Server Timeout" to "TiKV server timeout":

```markdown
The complete error message: `ERROR 9002 (HY000): TiKV Server Timeout`
```

Reply:
Thanks for the review.Updated.